examples/version-migration/schema-upgrade/tests/index.tsx

```diff
@@ -43,8 +44,14 @@ window["migrators"] = [];
 export async function createContainerAndRenderInElement(element: HTMLDivElement) {
 	const searchParams = new URLSearchParams(location.search);
 	const testMode = searchParams.get("testMode") !== null;
+	const configProvider = (settings: Record<string, ConfigTypes>): IConfigProviderBase => ({
+		getRawConfig: (name: string): ConfigTypes => settings[name],
+	});
 	const modelLoader = new SessionStorageModelLoader<IInventoryListAppModel>(
 		new DemoCodeLoader(testMode),
+		undefined, // logger
+		// To be removed ADO:5465
```

Comment:
+ @ChumpChief as FYI.

Reply:
I will notify all code owners with the appropriate bugs.

Reply:
Rather than plumb all this through, can you just have the test click the "add item" button to flip it into write mode?  This would be the more realistic scenario anyway.

Reply:
@ChumpChief this is the second iteration of this effort, as the first run by Curtis uncovered some other tests which were taken care of. In the meantime, new failures popped up, so we need to do this again on a test-by-test basis, hence the reason I opened the bugs and was planning on following up with the test owners to clarify the semantics of the test.

Reply:
I'll implement your suggestion for this particular test right now. Thanks for confirming there's no actual dependency on the container 'mode'

Reply:
Thanks!  Yeah I think you're just hitting an infinite wait for the summarizer to spawn (I assume it won't spawn for read-only connections).  So clicking the button before the summarizer await is prob sufficient.